### PR TITLE
Add summary bar for accounts view

### PR DESCRIPTION
**Pull request by vitaliyworks**

### CODE REVIEW

File: packages/page-accounts/src/Accounts/Account.tsx [hunk 1]
```diff
@@ -118,15 +132,24 @@ function Account ({ account: { address, meta }, className = '', delegation, filt
...
+      const balance: AccountBalance = {
+        bonded: stakingInfo?.stakingLedger.active.unwrap() ?? BN_ZERO,
+        locked: balancesAll.lockedBalance,
+        redeemable: stakingInfo?.redeemable ?? BN_ZERO,
+        total: balancesAll.freeBalance.add(balancesAll.reservedBalance),
+        transferrable: balancesAll.availableBalance,
+        unbonding: calcUnbonding(stakingInfo)
+      };
+
+      setBalance(address, balance);
```
Comment (jacogr) [inline packages/page-accounts/src/Accounts/Account.tsx:144]:
```suggestion
      setBalance(address, {
        bonded: stakingInfo?.stakingLedger.active.unwrap() ?? BN_ZERO,
        locked: balancesAll.lockedBalance,
        redeemable: stakingInfo?.redeemable ?? BN_ZERO,
        total: balancesAll.freeBalance.add(balancesAll.reservedBalance),
        transferrable: balancesAll.availableBalance,
        unbonding: calcUnbonding(stakingInfo)
      });
```

Why not simply the above?

File: packages/page-accounts/src/Accounts/Summary.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+function Summary ({ balance, className }: Props) {
+  const { t } = useTranslation();
+
+  return (
+    <SummaryBox className={className}>
+      {balance?.total.gtn(0) &&
+        <CardSummary label={t<string>('total balance')}>
+          <FormatBalance value={balance?.total} />
```
Comment (jacogr) [inline packages/page-accounts/src/Accounts/Summary.tsx:24]:
```suggestion
          <FormatBalance value={balance.total} />
```

Here (and below), we have already determined balance exists.

Actually the `balance?.` is quite heavy (read: horribly Babel-wise), so it is even better to do something like -

```js
<SummaryBox ...>
  {balance && (
    <>
      {balance.total.gt(BN_ZERO) && (
        <CardSummary ... />
      )}
    </>
  )}
</SummaryBox>
 ``

File: packages/page-accounts/src/Accounts/index.tsx [hunk 3]
```diff
@@ -228,10 +228,12 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
       </Button.Group>
       <BannerExtension />
       <BannerClaims />
+
+      {summary}
+
```
Comment (jacogr) [inline packages/page-accounts/src/Accounts/index.tsx:233]:
```suggestion
      <Summary balance={balances.summary} />
```

The Summary component should be React.memo-ed itself.

File: packages/page-accounts/src/Accounts/Account.tsx (hunk 1 above)
Comment (vitaliyworks) [reply packages/page-accounts/src/Accounts/Account.tsx:144]: :shrug: above does look better

File: packages/page-accounts/src/Accounts/Summary.tsx (hunk 2 above)
Comment (vitaliyworks) [reply packages/page-accounts/src/Accounts/Summary.tsx]: Done (implemented second option)

File: packages/page-accounts/src/Accounts/index.tsx (hunk 3 above)
Comment (vitaliyworks) [reply packages/page-accounts/src/Accounts/index.tsx]: Done

**Review by jacogr — APPROVED** [review]
Seems sane. tyvm.

**Pull request merged**